packages/account/src/Sections/Verification/ProofOfIdentity/onfido-sdk-view.tsx

```diff
@@ -76,28 +76,31 @@ const OnfidoSdkView = ({
                     />
                 </CSSTransition>
             </div>
-            {is_onfido_disabled && is_onfido_initialized && (
-                <HintBox
-                    className='onfido-container__info-message'
-                    icon='IcInfoBlue'
-                    icon_height={16}
-                    icon_width={16}
-                    message={
-                        <Text as='p' size='xxxs'>
-                            <Localize i18n_default_text='Hit the checkbox above to choose your document.' />
-                        </Text>
-                    }
-                    is_info
+            <section>
+                {is_onfido_disabled && is_onfido_initialized && (
```

Comment:
Why is this change needed?

Reply:
The Onfido sdk section is injected to the div where the id `onfido_element_id` is provided. But there was a requirement to overlap a banner on top of the sdk div. So rather than placing using positions, I have wrapped the div with a section enclosing the banner and the onfido sdk ui